x-pack/plugins/cloud_security_posture/common/types/rules/v3.ts

```diff
@@ -6,44 +6,17 @@
  */
 
 import { schema, TypeOf } from '@kbn/config-schema';
-import { CSPM_POLICY_TEMPLATE, KSPM_POLICY_TEMPLATE } from '../../constants';
+
+import { cspBenchmarkRuleMetadataSchema } from '@kbn/cloud-security-posture-common/schema';
```

Comment:
I still think we need to move the whole `common/types` with all the versions into our common package. Otherwise the versioning won't work for us. Imagine you need to make a breaking change to the schema. If you do it in the `cspBenchmarkRuleMetadataSchema` in the package you would break the versioning as it is imported here in the v3. To do it properly you need to introduce the new version of the schema, the same way as we migrated `cspBenchmarkRuleMetadataSchema` between v2 and v3.
I'm not sure if we can reduce the number of changes caused by moving the whole versioned types and schemas, but without it we are kind of breaking the versioning

Reply:
again to not make the change too large, maybe it's ok to proceed with this PR but then create a new PR right away moving the versioned types and schemas to our shared package so that we don't risk